apps/css-workshop/src/pages/tag.astro

```diff
@@ -19,433 +20,141 @@ import Layout from './_layout.astro';
-        <button
-          class='iui-button iui-tag-button'
-          data-iui-variant='borderless'
-          data-iui-size='small'
-          aria-label='Delete very very long tag'
-          type='button'
-        >
-          <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-        </button>
-      </span><span class='iui-tag'>
-        <span class='iui-tag-label' title='Tag'>Tag</span>
-        <button
-          class='iui-button iui-tag-button'
-          data-iui-variant='borderless'
-          data-iui-size='small'
-          aria-label='Delete tag'
-          type='button'
-        >
-          <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-        </button>
-      </span><span class='iui-tag'>
-        <span class='iui-tag-label' title='Long tag'>Long tag</span>
-        <button
-          class='iui-button iui-tag-button'
-          data-iui-variant='borderless'
-          data-iui-size='small'
-          aria-label='Delete long tag'
-          type='button'
-        >
-          <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-        </button>
-      </span><span class='iui-tag'>
-        <span class='iui-tag-label' title='Very long tag'>Very long tag</span>
-        <button
-          class='iui-button iui-tag-button'
-          data-iui-variant='borderless'
-          data-iui-size='small'
-          aria-label='Delete very long tag'
-          type='button'
-        >
-          <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-        </button>
-      </span><span class='iui-tag'>
-        <span class='iui-tag-label' title='Very very long tag'>Very very long tag</span>
-        <button
-          class='iui-button iui-tag-button'
-          data-iui-variant='borderless'
-          data-iui-size='small'
-          aria-label='Delete very very long tag'
-          type='button'
-        >
-          <svg-close-small class='iui-button-icon' aria-hidden='true'></svg-close-small>
-        </button>
-      </span>
+      <Tag_>Tag</Tag_>
+      <Tag_>Long tag</Tag_>
+      <Tag_>Very long tag</Tag_>
+      <Tag_>Very very long tag</Tag_>
+      <Tag_>Tag</Tag_>
+      <Tag_>Long tag</Tag_>
+      <Tag_>Very long tag</Tag_>
+      <Tag_>Very very long tag</Tag_>
+      <Tag_>Tag</Tag_>
+      <Tag_>Long tag</Tag_>
+      <Tag_>Very long tag</Tag_>
+      <Tag_>Very very long tag</Tag_>
     </div>
   </section>
 
   <hr />
-
   <h2>Basic tag containers</h2>
   <section id='demo-basic'>
     <div class='iui-tag-container'>
-      <span class='iui-tag-basic'>tags</span><span class='iui-tag-basic'>go</span><span
-        class='iui-tag-basic'>here</span
-      ><span class='iui-tag-basic'>tags</span><span class='iui-tag-basic'>go</span><span
-        class='iui-tag-basic'>here</span
-      >
+      <Tag_ variant='basic' 
+        >tags</Tag_><Tag_ variant='basic' 
```

Comment:
this is the specific formatting that prettier isn't too fond of

Reply:
Why is this formatting only being used in some of the tag containers? For example, the container directly above this one does not use this formatting.

Reply:
oh that was an accident. I thought I had committed those changes